_pages/404.md

```diff
@@ -4,6 +4,45 @@ title: 404 page not found!
 description: >
     404 page not found
 flow:
-    - row: main_content_row
+    - row: container_row
+      sections:
+       - format: title
+         title_content:
+            size: h1
+            text: 404 Page Not Found
```

Comment:
@kylekirkby This can be just "Page not found".

Reply:
Done.